cpp/arcticdb/version/local_versioned_engine.hpp

```diff
@@ -400,6 +394,7 @@ class LocalVersionedEngine : public VersionedEngine {
     }
     void _test_set_store(std::shared_ptr<Store> store);
     std::shared_ptr<VersionMap> _test_get_version_map();
+    SymbolList& _test_get_symbol_list();
```

Comment:
Return by value to avoid confusion if this is just for testing?

Reply:
For the testing only in the other repo

Reply:
Agree with Alex, returning by reference is unnecessarily dangerous for test code

Reply:
It is the last resort for https://github.com/man-group/arcticdb-enterprise/blob/2af2e32dc3321b28f03a9167bc0d40092f22e7a8/cpp/arcticdb-enterprise/background_delete/test/test_deletion.cpp#L74

Reply:
Actually, why do we need this at all? Just call `symbol_list()` directly from the test?

Reply:
The dependency is removed by turning off the `symbol_list` in the test